code/components/gta-core-five/src/CrashFixes.TaskGeneralSweepClip.cpp

```diff
@@ -0,0 +1,34 @@
+#include <StdInc.h>
+
+#include <Hooking.h>
+
+static bool (*g_origCTaskGeneralSweep__Unk)(void* task, void* clip, float* unk1, float* unk2, float* unk3);
+static bool CTaskGeneralSweep__Unk(void* task, void* clip, float* unk1, float* unk2, float* unk3)
+{
+	if (!clip)
+		return false;
+
+	return g_origCTaskGeneralSweep__Unk(task, clip, unk1, unk2, unk3);
+}
+
+static HookFunction hookFunction([]
+{
+	auto patterns = hook::pattern("48 8B ? 48 8B CB 48 89 44 24 20 E8 ? ? ? ? 84").count_hint(5);
```

Comment:
We can't be sure that occurences will be always aligned as we need. Can we switch to just finding 3 needed targets with unique pattern then?
`48 8B D5 48 8B CB 48 89 44 24 ? E8 ? ? ? ? 84 C0 75`
`48 8B D5 48 8B CB 48 89 44 24 ? E8 ? ? ? ? 84 C0 74`
`48 8B D7 48 8B CB 48 89 44 24 ? E8 ? ? ? ? 84 C0`

Reply:
You don't need to do anything, I've fixed it in the merge process.

Reply:
Checked across multiple builds and wasn't sure if that could happen, will keep that in mind, thanks :)